services/controllerhost/event_handlers.go

```diff
@@ -99,6 +99,16 @@ type (
 		storeIDs []string
 	}
 
+	// StoreRemoteExtentReplicatorDownEvent is triggered
+	// for an extent when the primary store host that's
+	// responsible for replicating that extent from origin
+	// zone is down
+	StoreRemoteExtentReplicatorDownEvent struct {
```

Comment:
is RemoteExtentPrimaryStoreDownEvent more clear? Replicator makes me this is about the replicator role